orc8r/cloud/go/services/bootstrapper/servicers/registration/cloud_registration.go

```diff
@@ -0,0 +1,149 @@
+package registration
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"magma/orc8r/lib/go/protos"
+
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	nonceLength = 12
+	timeoutDuration = 30 * time.Minute
```

Comment:
It's an arbitrary amount of token timeout time I believe? @hcgatewood might have more context

Reply:
Can I add a TODO and add this in another PR?

Reply:
I'd prefer we add it in this PR so we don't lose track of it -- I'd argue configurability of this value is integral to the design of this feature

Reply:
I'm putting the `bootstrapper.yml` in the last PR in the stack: [Add Servicers to Bootstrapper Service PR](https://github.com/reginawang3495/magma/pull/7) because I don't want to rebase a bunch of times